modules/t/bigInteract.t

```diff
@@ -0,0 +1,47 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2018] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Test::More;
+use Bio::EnsEMBL::IO::Parser::BigBed;
+
+## TEST FOR UCSC bigInteract FORMAT
+## which is a custom bigBed format with specific AutoSQL fields
+
+######################################################
+## Test 1
+######################################################
+my $parser = Bio::EnsEMBL::IO::Parser::BigBed->open('modules/t/input/data_interact.bb');
```

Comment:
Harcoded file paths are to be avoided, can you please replace this line with
`use FindBin;`
`my $test_file = $FindBin::Bin . '/input/data_interact.bb';`
`my $parser = Bio::EnsEMBL::IO::Parser::BigBed->open($test_file);`

Reply:
Thanks! I saw that change this morning, then forgot to add it to the new test!